src/test/java/org/fundacionjala/at13/spaceinvaders/bulletTest.java

```diff
@@ -261,4 +302,7 @@ public void itShouldReturnBulletToPosition0X0YWhenItGetYLimitForMovingBulletUp()
         bulletAlien.movingBulletUp();
         assertEquals(0, bulletAlien.getPositionY());
     }
+
+
+
```

Comment:
these spaces should be removed in the code

Reply:
ok